### PR TITLE
Ins 30373

### PR DESCRIPTION
**Pull request by RefaelAdi**
1. Entered the SaaS docs and updated the plugins readme file accordingly.
2. Fixed incorrect URLs.
3. Organized the plug-ins list alphabetically.

### CODE REVIEW

File: docs/Guardium_Insights/SaaS_1.0/UC_Configuration_GI.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    
+      ii. **Description**: Enter a description for the connection.
+    
+    b. Click **Next**.
+    
+    c. In the Build pipeline page, use the **Choose input plug-in** menu to select your input plug-in. Then select a filter plug-in by using the **Choose a filter plug-in** menu.
+    
+    d. Click **Next**.
+    
+    e. Follow the instructions according to the input plug-in type you selected in <em>**step c**</em>.
```
Comment (AlexaNahum) [inline docs/Guardium_Insights/SaaS_1.0/UC_Configuration_GI.md:47]:
```suggestion
    e. Follow the instructions according to the input plug-in type you selected in **step c**.
```

File: docs/Guardium_Insights/SaaS_1.0/UC_Configuration_GI.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+ In the **Additional info** page, specify the following details for the connection you want to create:
+
+*   **Connection string** - the JDBC connection string, for example:
+   ```
+   jdbc:sqlserver://<server_name>:<port_number>;databaseName=<database_name>
+   ```
+**NOTE** Do not enter the database user name and password in the connection string, as they are supplied in the **JDBC user** and **Password** fields mentioned in previous steps.
+
+*   **Statement** - the statement setting determines which audit tables the SELECT query calls for the audit logs. In the Guardium UI, the **Statement*** is divided into three parts to enhance clarity and ease of use: `SELECT` for choosing columns, `FROM` for specifying tables, and `WHERE` for adding filter conditions.
```
Comment (AlexaNahum) [inline docs/Guardium_Insights/SaaS_1.0/UC_Configuration_GI.md:59]:
```suggestion
*   **Statement** - the statement setting determines which audit tables the `SELECT` query calls for the audit logs. In the Guardium UI, the **Statement*** is divided into three parts to enhance clarity and ease of use: `SELECT` for choosing columns, `FROM` for specifying tables, and `WHERE` for adding filter conditions.
```

File: docs/Guardium_Insights/SaaS_1.0/UC_Configuration_GI.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+   for example:
+   ```
+   ./create_certificates.sh /path/to/store datasource.server.dns.com
+   ```
+
+4. Copy `filebeatCA.crt` to your local system.
+
+### Configuring a Filebeat connection:
+
+1. Exectue steps 1-4 from the [main procedure](../SaaS_1.0/UC_Configuration_GI.md#procedure) . For the data source, select **MongoDB**. For the data source enviornment type, select **On-premises**. For the input plug-in, choose **Filebeat**.
```
Comment (AlexaNahum) [inline docs/Guardium_Insights/SaaS_1.0/UC_Configuration_GI.md:94]:
```suggestion
1. Execute steps 1-4 from the [main procedure](../SaaS_1.0/UC_Configuration_GI.md#procedure) . For the data source, select **MongoDB**. For the data source environment type, select **On-premises**. For the input plug-in, choose **Filebeat**.
```

File: docs/Guardium_Insights/SaaS_1.0/UC_Configuration_GI.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+In the **Additional info** page, specify the details of the connection you want to create:
+
+* **Event hub connections** - specify the list of connection strings that identifies the Event Hubs to be read. Connection strings include the `EntityPath` for the Event Hub.
+
+* **Enrollment ID** - Azure enrollment ID: Unique subscription identifier for billing and resource management.
+ ## Configuring Filebeat to forward audit logs to Guardium
+    
+    
+  ## Procedure
+prerequisites:
```
Comment (AlexaNahum) [inline docs/Guardium_Insights/SaaS_1.0/UC_Configuration_GI.md:140]:
```suggestion
### Prerequisites:
```

File: docs/Guardium_Insights/SaaS_1.0/UC_Configuration_GI.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+1.  Open the Filebeat configuration (filebeat.yml) usually located in `/etc/filebeat/filebeat.yml`.
+    
+2.  Locate the `tags` section and enter the data source tag. For example, `tags: ["any-mongodb"]`.
+    
+3.  Locate the `output.logstash` section and add an entry for Guardium Insights:
+    ```
+    # The Logstash hosts
+    hosts: ["<hostname-URL>:443"]
+    ```
+**NOTE**: In Guardium Insights, whenever using plug-ins that are based on Filebeat as a data shipper, the configured port should be 443. Guardium Insights will map this to an internal port
```
Comment (AlexaNahum) [inline docs/Guardium_Insights/SaaS_1.0/UC_Configuration_GI.md:155]:
```suggestion
**NOTE**: In Guardium Insights, whenever you are using plug-ins that are based on Filebeat as a data shipper, the configured port should be 443. Guardium Insights will map this to an internal port
```

File: docs/available_plugins.md [hunk 6]
```diff
@@ -1,54 +1,54 @@
...
-* [ProgressDB](../filter-plugin/logstash-filter-progressdb-guardium/README.md) (Future GI releases)
+* [Yugabyte](../filter-plugin/logstash-filter-yugabyte-guardium/README.md)
 
 ## Developing Plug-ins
 Users can develop their own universal connector plug-ins, if needed, and contribute them back to the open source project, if desired.
 
 [Here](../docs/Guardium%20Data%20Protection/developing_plugins_gdp.md) is a guide for developing new plug-ins for Guardium Data Protection.
 
-[Here](../docs/Guardium%20Insights/3.2.x/developing_plugins_gi.md) is a guide for developing new plug-ins for Guardium Insights.
+[Here](../docs/Guardium%20Insights/3.2.x/developing_plugins_gi.md) is a guide for developing new plug-ins for Guardium Insights (for 3.2.x versions only).
```
Comment (AlexaNahum) [inline docs/available_plugins.md:54]:
```suggestion
[Here](../docs/Guardium%20Insights/3.2.x/developing_plugins_gi.md) is a guide for developing new plug-ins for Guardium Insights (version 3.2.x only).
```

File: docs/available_plugins.md [hunk 7]
```diff
@@ -1,54 +1,54 @@
...
+* [Neptune](../filter-plugin/logstash-filter-neptune-aws-guardium/README.md)
+* [Oracle](../filter-plugin/logstash-filter-oua-guardium/README.md) (Future GI releases)
+* [PostgreSQL](../filter-plugin/logstash-filter-onPremPostgres-guardium/README.md)
 * [SAP HANA](../filter-plugin/logstash-filter-saphana-guardium/README.md)
-* [Snowflake](https://github.com/infoinsights/guardium-snowflake-uc-filter) (soon: GI 3.3)
+* [Snowflake](https://github.com/infoinsights/guardium-snowflake-uc-filter)
 * [Teradata](../filter-plugin/logstash-filter-teradatadb-guardium/README.md) (Future GI releases)
-* [Yugabyte](../filter-plugin/logstash-filter-yugabyte-guardium/README.md) (soon: GI 3.3)
-* [ProgressDB](../filter-plugin/logstash-filter-progressdb-guardium/README.md) (Future GI releases)
+* [Yugabyte](../filter-plugin/logstash-filter-yugabyte-guardium/README.md)
```
Comment (AlexaNahum) [inline docs/available_plugins.md:48]: it seems like some plug-ins were removed from the list and not replaced (ProgressDB, MSSQL.. I'm assuming that's on purpose?)

**Review by AlexaNahum — APPROVED** [review]

File: filter-plugin/logstash-filter-snowflake-guardium/README.md [hunk 8]
```diff
@@ -1,4 +1,11 @@
 # Guardium Snowflake filter plug-in
+* * Environment: AWS, Azure, GCP
+* Supported Guardium versions:
+    * Guardium Data Protection: 11.4 and above
+        * Supported inputs:
+            * JDBC (pull)
+    * Guardium Insights: 3.2.6 and above
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-snowflake-guardium/README.md:7]: I'm not sure we should write 3.2.6.
It is good enough for a POC, but we still need to discuss how we want to handle load balancing

File: docs/available_plugins.md [hunk 9]
```diff
@@ -1,54 +1,54 @@
 # Available Plug-ins
```
Comment (ofer-haim) [inline docs/available_plugins.md:1]: Need to review this page again after the merge to verify all the links are still working

**Review by ofer-haim — APPROVED** [review]
Once this is merged we need to:

1. Verify links in available_plugins.md page again
2. Let PM review this as well
3. Remove everything we can from the internal repo so it won't be maintained in 2 different places

File: filter-plugin/logstash-filter-snowflake-guardium/README.md (hunk 8 above)
Comment (RefaelAdi) [reply filter-plugin/logstash-filter-snowflake-guardium/README.md:7]: I'll change it to SaaS 1.0

File: filter-plugin/logstash-filter-aurora-mysql-guardium/README.md [hunk 10]
```diff
@@ -5,6 +5,7 @@
 * Supported inputs: CloudWatch (pull)
 * Supported Guardium versions:
 	* Guardium Data Protection: 11.4 and above
+    * Guardium Insights: SaaS 1.0
```
Comment (hadarkorny) [inline filter-plugin/logstash-filter-aurora-mysql-guardium/README.md:8]:
```suggestion
* Supported Guardium versions:
	* Guardium Data Protection: 11.4 and above
	* Guardium Insights: SaaS 1.0
```

File: filter-plugin/logstash-filter-azure-sql-guardium/README.md [hunk 11]
```diff
@@ -5,6 +5,7 @@
 * Supported inputs: JDBC (pull)
 * Supported Guardium versions:
 	* Guardium Data Protection: 11.4 and above
+    * Guardium Insights: SaaS 1.0
```
Comment (hadarkorny) [inline filter-plugin/logstash-filter-azure-sql-guardium/README.md:8]:
```suggestion
	* Guardium Data Protection: 11.4 and above
	* Guardium Insights: SaaS 1.0
```

File: filter-plugin/logstash-filter-cassandra-guardium/README.md [hunk 12]
```diff
@@ -5,6 +5,7 @@
 * Supported inputs: Filebeat (push)
 * Supported Guardium versions:
 	* Guardium Data Protection: 11.4 and above
+    * Guardium Insights: SaaS 1.0
```
Comment (hadarkorny) [inline filter-plugin/logstash-filter-cassandra-guardium/README.md:8]:
```suggestion
	* Guardium Data Protection: 11.4 and above
	* Guardium Insights: SaaS 1.0
```

File: filter-plugin/logstash-filter-mssql-guardium/README.md [hunk 13]
```diff
@@ -5,7 +5,7 @@
 * Supported inputs: JDBC (pull)
 * Supported Guardium versions:
 	* Guardium Data Protection: 11.4 and above
-    * Guardium Insights: coming soon
+    * Guardium Insights: SaaS 1.0
```
Comment (hadarkorny) [inline filter-plugin/logstash-filter-mssql-guardium/README.md:8]:
```suggestion
	* Guardium Insights: SaaS 1.0
```

File: filter-plugin/logstash-filter-mysql-guardium/README.md [hunk 14]
```diff
@@ -8,8 +8,6 @@
          * Syslog (push)
          * Filebeat (push)
    * Guardium Insights: 3.2 and above
-      * Supported inputs:
-         * Filebeat (push)
```
Comment (hadarkorny) [inline filter-plugin/logstash-filter-mysql-guardium/README.md:12]: why delete? if there is a new supported input I think we rather add it

File: filter-plugin/logstash-filter-snowflake-guardium/README.md [hunk 15]
```diff
@@ -1,4 +1,11 @@
 # Guardium Snowflake filter plug-in
+* * Environment: AWS, Azure, GCP
+* Supported Guardium versions:
+    * Guardium Data Protection: 11.4 and above
+        * Supported inputs:
+            * JDBC (pull)
+    * Guardium Insights: SaaS 1.0
```
Comment (hadarkorny) [inline filter-plugin/logstash-filter-snowflake-guardium/README.md:7]:
```suggestion
* Environment: AWS, Azure, GCP
* Supported Guardium versions:
    * Guardium Data Protection: 11.4 and above
        * Supported inputs:
            * JDBC (pull)
    * Guardium Insights: SaaS 1.0
```

**Review by hadarkorny — APPROVED** [review]

File: filter-plugin/logstash-filter-snowflake-guardium/README.md (hunk 15 above)
Comment (RefaelAdi) [reply filter-plugin/logstash-filter-snowflake-guardium/README.md]: updated

File: filter-plugin/logstash-filter-mysql-guardium/README.md (hunk 14 above)
Comment (RefaelAdi) [reply filter-plugin/logstash-filter-mysql-guardium/README.md:12]: fixed

**Pull request merged**